app/src/main/java/org/coepi/android/cen/CenRepo.kt

```diff
@@ -67,7 +67,9 @@ class CenRepo(private val cenApi: CENApi, private val cenDao: RealmCenDao, priva
             cenKeyTimestamp = curTimestamp
             cenkeyDao.insert(CenKey(cenKey, cenKeyTimestamp))
         }
-        CEN.onNext(generateCEN(cenKey, curTimestamp))
+        val cen= generateCEN(cenKey, curTimestamp);
```

Comment:
Can you please use autoformat. It's just pressing a shortcut when you're finished editing. https://jimbaca.com/auto-format-code-in-android-studio/

Reply:
added 1 space
-        val cen= generateCEN(cenKey, curTimestamp);
+        val cen = generateCEN(cenKey, curTimestamp);